src/graph_notebook/notebooks/01-Getting-Started/01-About-the-Neptune-Notebook.ipynb

```diff
@@ -271,9 +271,9 @@
     "\n",
     "Curious about the business problems can be solved with graph? Check out these sample application notebooks for some inspiration.\n",
     "\n",
-    "[Introduction to Fraud Graphs](../Introduction-to-Fraud-Graphs.ipynb)\n",
+    "[Introduction to Fraud Graphs](../03-Sample-Applications/01-Fraud-Graphs/01-Building-a-Fraud-Graph-Application.ipynb)\n",
     "\n",
-    "[Introduction to Knowledge Graphs](../Introduction-to-Knowledge-Graphs.ipynb)\n",
+    "[Introduction to Knowledge Graphs](../03-Sample-Applications/02-Knowledge-Graphs/01-Building-a-Knowledge-Graph-Application.ipynb)\n",
     "\n",
     "[Introduction to Identity Graphs](../Introduction-to-Identity-Graphs.ipynb)"
```

Comment:
Need to change this to: "[Introduction to Identity Graphs](../03-Sample-Applications/03-Identity-Graphs/01-Building-an-Identity-Graph-Application.ipynb)"